tests/resolvers/Mutation/acceptAdmin.spec.ts

```diff
@@ -36,22 +33,6 @@ afterEach(() => {
 });
 
 describe("resolvers -> Mutation -> acceptAdmin", () => {
-  it(`throws Error if user with _id === context.userId is not a SUPERADMIN`, async () => {
```

Comment:
Did you remove this check because you're already testing the edge case in `superAdminCheck.spec.ts` test file?